distributed/tests/test_scheduler.py

```diff
@@ -596,6 +596,19 @@ def test_coerce_address():
         yield [w.close() for w in [a, b, c]]
 
 
+@gen_test()
+def test_config_stealing():
```

Comment:
Is it preferred to use `@gen_test()` or `@pytest.mark.asyncio`? There are a few of both here

Reply:
I'm not sure if there's a reason to prefer one over the other (although others may have an opinion here). FWIW there are ~3x more occurrence of `pytest.mark.asyncio` than `gen_test`, so maybe `pytest.mark.asyncio` is preferred

Reply:
Makes sense. Rewrote as suggested using `pytest.mark.asyncio`. PTAL 🙂